syncer/checkpoint.go

```diff
@@ -480,15 +481,14 @@ func (cp *RemoteCheckPoint) prepare(tctx *tcontext.Context) error {
 }
 
 func (cp *RemoteCheckPoint) createSchema(tctx *tcontext.Context) error {
-	sql2 := fmt.Sprintf("CREATE SCHEMA IF NOT EXISTS `%s`", cp.schema)
+	sql2 := fmt.Sprintf("CREATE SCHEMA IF NOT EXISTS %s", dbutil.ColumnName(cp.schema))
```

Comment:
How about quoting it when init it?

Reply:
https://github.com/pingcap/dm/pull/840/commits/54db3d8b21ab19943448e5a1de07ced2a533f56b